data/deep missions.txt

```diff
@@ -351,83 +360,86 @@ mission "Deep: Remnant 0"
-			`	"Thank you for buying these keystones for us. It may take us a month or two to fully study them. We will make sure to contact you when we are in need of your assistance again."`
+		conversation # Maria will be introduced in Deep: TMBR. She could be postdoc / partner scientist etc.
+			`Upon returning to <planet>, you learn that Ivan has left for his research lab on Valhalla, but his research assistant Maria is waiting to retrieve the keystones. She hands you <payment>, enough to pay for half of the cost of the stones.`
+			`	"Thank you for buying these keystones for us. It may take us a month or two to fully study them. We will contact you when we are in need of your assistance again."`
 			
 event "deep: keystone research"
 
 mission "Deep: Remnant: Keystone Research"
 	landing
 	name `Travel to <planet>`
-	description `Meet up with Ivan on <destination> to learn about how his research on the Quantum Keystones has revealed.`
+	description `Meet up with Ivan on <destination> to learn about his research on the Quantum Keystones you brought him.`
 	source 
 		government Republic "Free Worlds" Syndicate Neutral Independent
 	destination Valhalla
 	to offer
 		has "event: deep: keystone research"
 	on offer
 		dialog
-			`You recieve a message from Ivan, the scientists working on the spacial anomaly in Terminus. "Captain, we have finished studying the Quantum Keystones, and they do in fact allow access through the wormhole. Please contact me on <destination> if you wish to continue helping us with our research."`
+			`You receive a message from Ivan, the scientist working on the spatial anomaly in Terminus. "Captain, we have finished studying the Quantum Keystones, and they do in fact allow access through the wormhole. Please meet with me on <destination> if you wish to continue helping us with our research."`
 
 mission "Deep: Remnant 1A"
 	landing
 	name `Exploring the Unknown`
-	description `Explore the region of space beyond the wormhole in Terminus. Return to <destination> after exploring as much as you can.`
+	description `Explore the region of space beyond the Terminus wormhole. Return to <destination> after exploring everything you can reach.`
 	source Valhalla
 	waypoint Terminus
 	to offer
 		has "Deep: Remnant: Keystone Research: done"
 		not "knew of stones before scientists"
 	on offer
 		conversation
-			`You contact Ivan after you land and he sends you directions to his research lab, which is located on the outskirts of the spaceport city. When you arrive you are greeted by Ivan, Maria, and a number of other scientists on the team. Ivan begins to inform you on what their research has revealed.`
+			`You contact Ivan after you land and he sends you directions to his research lab on the outskirts of the nearby city. When you arrive you are greeted by Ivan, Maria, and a number of other scientists on the team, and Ivan eagerly informs you on what their research has revealed.`
```

Comment:
Spaceports seem to be part of cities, so I wouldn't exactly describe the city as being "nearby;" you're already in the city once you've landed in the spaceport.